lesson20/homework.py

```diff
@@ -0,0 +1,51 @@
+from pathlib import Path
+import logging
+from datetime import datetime
+
+log_file_path = Path(__file__).parent / "hb_test.log"
```

Comment:
```suggestion
log_file_path = Path(__file__).parent / "hb_test.log"
```
Файлу логу я не бачу (бо він в ігнор) але, треба привести лог в опису PR